server/views/serviceProviderReferrals/postSessionFeedbackCheckAnswers.njk

```diff
@@ -1,34 +1,10 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 
 {% extends "./actionPlan/actionPlanFormTemplate.njk" %}
 
 {% block formSection %}
-  {{ govukSummaryList(summaryListArgs) }}
```

Comment:
You seem to have lost the summary list here — it's not in the new partial.

Reply:
This was a real pain due to the way this branch was committed and this kept disappearing and reappearing at various points in history during my many rebases. I think I've managed to solve it now though!